tests/providers/test_address.py

```diff
@@ -9,6 +9,7 @@
 from faker.providers.address.cs_CZ import Provider as CsCzAddressProvider
 from faker.providers.address.de_AT import Provider as DeAtAddressProvider
 from faker.providers.address.de_DE import Provider as DeDeAddressProvider
+from faker.providers.address.Dk_Dk import Provider as DkDkAddressProvider
```

Comment:
This import has the wrong capitalization. Is should be `faker.providers.address.dk_DK`

```suggestion
from faker.providers.address.dk_DK import Provider as DkDkAddressProvider
```